5.5/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM openshift/base-centos7
+
+# This image provides an Apache+PHP environment for running PHP
+# applications.
+
+MAINTAINER Ben Parees <bparees@redhat.com>
+
+# Image metadata
+ENV PHP_VERSION           5.5
+ENV IMAGE_DESCRIPTION     PHP 5.5
+ENV IMAGE_TAGS            php,php55
+ENV IMAGE_EXPOSE_SERVICES 8080:http
+#ENV PHPRC /opt/openshift/etc/php.ini
+#ENV PHP_INI_SCAN_DIR /opt/openshift/etc/php.d
+
+# Install Apache httpd and PHP
+RUN yum install -y \
+    https://www.softwarecollections.org/en/scls/rhscl/httpd24/epel-7-x86_64/download/rhscl-httpd24-epel-7-x86_64.noarch.rpm \
+    https://www.softwarecollections.org/en/scls/rhscl/php55/epel-7-x86_64/download/rhscl-php55-epel-7-x86_64.noarch.rpm && \
+    yum install -y --setopt=tsflags=nodocs httpd24 php55 php55-php php55-php-mysqlnd php55-php-pgsql php55-php-bcmath php55-php-devel \
```

Comment:
i'd actually say it's a bit harder to track new packages since they'd just appear w/o us doing anything.

worse a package that was removed would not cause a build failure (not sure that's a likely occurrence) if we were just wildcarding.